dlutils/dataset/dataset.py

```diff
@@ -47,10 +51,17 @@ def parse_from_tfrecords(filename_pattern,
     # collect files.
     filenames = tf.data.Dataset.list_files(filename_pattern)
 
+    if balance_records:
+        load_record_fun = lambda x: tf.data.TFRecordDataset(x).repeat()
+    else:
+        load_record_fun = tf.data.TFRecordDataset
+
     # read from all with interleave...
-    dataset = filenames.interleave(tf.data.TFRecordDataset,
-                                   cycle_length=cycle_length,
-                                   num_parallel_calls=num_parallel_calls)
+    dataset = filenames.interleave(
+        load_record_fun,
+        # ~dataset = filenames.interleave(tf.data.TFRecordDataset,
```

Comment:
```suggestion
```

Can we get rid of this comment line? As it is now, with this indentation, it's more confusing than clarifying.

Reply:
yes that should have been gone